src/components/molecules/CategoryEdit/index.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <form @submit.prevent="editCategory">
+    <app-heading :level="1">カテゴリー管理</app-heading>
+
+    <div class="category-detail__back">
+      <app-router-link
+        class="category-management-edit__link"
+        block
+        underline
+        key-color
+        hover-opacity
+        to="/categories"
+      >
+        カテゴリー一覧へ戻る
+      </app-router-link>
+    </div>
+
+    <app-input
+      v-validate="'required'"
+      name="category"
+      type="text"
+      placeholder="更新するカテゴリー名を入力してください"
+      data-vv-as="カテゴリー名"
+      :error-messages="errors.collect('category')"
+      :value="category"
+      @update-value="$emit('update-value', $event)"
+    />
+    <app-button
+      class="category-management-edit__submit"
+      button-type="submit"
+      round
+      :disabled="disabled || !access.create"
```

Comment:
[imo]

以下が適切かと思います。

```suggestion
      :disabled="disabled || !access.edit"
```

Reply:
修正いたしました。